setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="scp-ingest-pipeline",
-    version="1.11.0",
+    version="1.11.1",
```

Comment:
[Current version](https://github.com/broadinstitute/scp-ingest-pipeline/releases) is 1.12.0, so this should be higher than that.